tests/high_performance_vm_test.go

```diff
@@ -39,42 +39,42 @@ var _ = Describe("High performance vm test", func() {
 		ktests.BeforeTestCleanup()
 	})
 
-	Context("Headless vm test", func() {
+	Context("[rfe_id:609, crit:medium, vendor:cnv-qe@redhat.com, level:component]Headless vm test", func() {
```

Comment:
There is no need to repeat information, if the rfe, criticality, test level & vendor are on the describe level and shouldn't be overridden, you can just remove it from here

Reply:
And please change the structure to
[Attr1][attr2] instead of [Attr1, attr2]